angularSTEP/src/app/profile-menu/profile-menu.component.scss

```diff
@@ -1,4 +1,5 @@
-$avatar-size: 30px;
+$avatar-size: 35px;
+$ripple-size: 60px;
```

Comment:
Out of curiosity, what does "ripple" mean here?

Reply:
I am using Angular Material Ripples, which make a wave like effect when you click a button. For the menu item, when you click the profile button, it will cause a ripple effect to take place in the background.